packages/docs/content/components/button.mdx

```diff
@@ -19,6 +19,8 @@ import { Alert, ArrowIcon, Button } from '@cmsgov/design-system';
   
 </Alert>
 
+Buttons are promises to the user; they must deliver the promise they offer by doing what the button says it will do.
```

Comment:
Should this go in the intro section in the header?

Reply:
Or perhaps in the guidance section if it's too much?

Reply:
Yeah good idea to move it to the guidance section - I'll do that!